docs/sources/setup-grafana/installation/debian/index.md

```diff
@@ -70,35 +72,7 @@ sudo apt-get install grafana
 sudo apt-get install grafana-enterprise
 ```
 
-### Install .deb package
-
-If you install the `.deb` package, then you will need to manually update Grafana for each new version.
-
-1. On the [Grafana download page](https://grafana.com/grafana/download), select the Grafana version you want to install.
-   - The most recent Grafana version is selected by default.
-   - The **Version** field displays only finished releases. If you want to install a beta version, click **Nightly Builds** and then select a version.
-1. Select an **Edition**.
-   - **Enterprise** - Recommended download. Functionally identical to the open source version, but includes features you can unlock with a license if you so choose.
-   - **Open Source** - Functionally identical to the Enterprise version, but you will need to download the Enterprise version if you want Enterprise features.
-1. Depending on which system you are running, click **Linux** or **ARM**.
-1. Copy and paste the code from the installation page into your command line and run. It follows the pattern shown below.
-
-```bash
-sudo apt-get install -y adduser
-wget <.deb package url>
-sudo dpkg -i grafana<edition>_<version>_amd64.deb
-```
-
-## Install from binary .tar.gz file
-
-Download the latest [`.tar.gz` file](https://grafana.com/grafana/download?platform=linux) and extract it. The files extract into a folder named after the Grafana version downloaded. This folder contains all files required to run Grafana. There are no init scripts or install scripts in this package.
-
-```bash
-wget <tar.gz package url>
-sudo tar -zxvf <tar.gz package>
-```
-
-## 2. Start the server
+## Start the Grafana server
```

Comment:
@DanCech, I recommend moving all start server content out of the installation topic and into a topic previously known as "Restart the Grafana server", which I have hijacked and am now calling "Start the Grafana Server". This PR includes a first pass at moving the Linux, Mac, and Windows instructions there.

However, these instructions must remain in this topic for now because there are links from the download page to this content. If you agree with this approach, let's coordinate on changing those links in the download page after this PR is merged. Then I'll remove the start server instructions from this topic.